CHANGELOG.md

```diff
@@ -100,6 +100,22 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
+## Unreleased
+
+### Changed
+
+- Remove usage of `kube-rbac-proxy` from `default` kustomization.
+  Its functionality of can be now achieved by using the new flag `--metrics-access-filter`
```

Comment:
Maybe `--metrics-access-protection` or `--protected-access-metrics` instead of proposed filter? WDYT?
But I see they refer to it as filter in `FilterProvider`, so it's for your own judge ;)

Reply:
Upstream field name was my motivation for this name. I'm fine changing this as long as others also think this would be better named this way.

Reply:
I'd say it's better to stick to the upstream naming than invent something else on our own.